vast.conf.example

```diff
@@ -1,447 +1,411 @@
+  #max-events: <infinity>
+
+  # Timeout after which batched table slices are forwarded.
+  batch-timeout = "10s"
+
+  # Read timoeut after which data is forwarded to the importer regardless of
+  # batching and table slices being unfinished.
+  read-timeout: "10s"
+
+  # Block until the importer forwarded all data.
+  blocking: false
+
+  # Upper bound for the size of a table slice. A value of 0 causes the
+  # batch-size to be unbounded, leaving control of batching to the
+  # import.batch-timeout option only.
+  batch-size = 100
+
+  # Encoding type of table slices (arrow or msgpack).
+  batch-encoding: arrow
+
+
+  # The `vast import csv` command imports data from CSVs with a known schema.
+  csv:
+    # The endpoint to listen on ("[host]:port/type").
+    #listen: <none>
+
+    # Path to file to read events from or "-" for stdin.
+    #read: "-"
+
+    # Treat the read option as a UNIX domain socket to connect to.
+    #uds: false
+
+    # Path to an alternate schema.
+    #schema-file: <none>
+
+    # An alternate schema as a string.
+    #schema: <none>
+
+  # The `vast import json` command imports data from JSONLs with a known schema.
+  json:
+    # For available options, see import.csv.
+
+  # The `vast import pcap` command imports PCAP logs.
+  pcap:
+    # Network interface to read packets from.
+    #interface: <none>
+
+    # Skip flow packets after this many bytes.
+    #cutoff: <infinity>
+
+    # Number of concurrent flows to track.
+    #max-flows: 1048576
+
+    # Maximum flow lifetime before eviction.
+    #max-flow-age: 60
+
+    # Flow table expiration interval.
+    #flow-expiry: 10
+
+    # Inverse factor by which to delay packets. For example, if 5, then for two
+    # packets spaced *t* seconds apart, the source will sleep for *t/5* seconds.
+    #pseudo-realtime-factor: 0
+
+    # Snapshot length in bytes.
+    #snaplen: 65535
+
+    # Disable computation of community id for every packet.
+    # disable-community-id: false
+
+    # For additionally available options, see import.csv.
+
+  # The `vast import suricata` command imports Suricata eve.json logs.
+  suricata:
+    # For available options, see import.csv.
+
+  # The `vast import syslog` command imports Syslog entries.
+  syslog:
+    # For available options, see import.csv.
+
+  # The `vast import test` command imports randomly generated events. Used for
+  # debugging and benchmarking only.
+  test:
+    # The PRNG seed.
+    seed: 0
+
+    # For additionally available options, see import.csv.
+
+  # The `vast import zeek` command imports Zeek logs.
+  zeek:
+    # For available options, see import.csv.
+
+# The `vast pivot` command extracts related events of a given type.
+pivot:
+  # The output format.
+  # format: "json"
+
+  # For additionally available options, see export.pcap.
+
+# The `vast status` command prints a JSON-formatted status summary of the node.
+status:
+  # No further configuration options are available. The system options apply.
```

Comment:
This is not true.

Reply:
I just took this from the default. How about we change this as we are restructuring the YAML file and streamline the sections? Then it becomes also easier to see how the transformation from CAF to YAML config took place.

Reply:
Alrighty.